problems/objects.js

```diff
@@ -187,15 +268,50 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let newStr = str.split(" ").join("").toLowerCase()
+  let count = { }
+  for(let i = 0; i < newStr.length; i++){
+    const el = newStr[i]
+    if(count[el] !== undefined){
+      count[el] += 1
+    } else {
+      count[el] = 1
+    }
+  } 
+    return count
+};
 
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
+const mostCommonElement = (array) => {
+  let obj = {};
```

Comment:
Think about splitting this part of your code into a helper function so that each function only does one thing.